source/_docs/automation/trigger.markdown

```diff
@@ -752,3 +783,4 @@ automation:
         - sensor.two
         - sensor.three
 ```
+For state and numeric_state triggers, you may use a [template](/docs/configuration/templating/) for configuring the `entity_id` property. This is useful for Blueprint developers to allow a selector to specify the entities for the trigger. The template will only be evaluated at configuration time. Only [limited templates](/docs/configuration/templating/#limited-templates) are allowed.
```

Comment:
This is not useful for Blueprints, as there is a selector for that available and can be used directly without templating.

```suggestion

For state and numeric_state triggers, you may use a [template](/docs/configuration/templating/) for configuring the `entity_id` property. The template will only be evaluated at configuration time. Only [limited templates](/docs/configuration/templating/#limited-templates) are allowed.
```

Reply:
Changed to:
+For state and numeric_state triggers, you may use a [template](/docs/configuration/templating/) for configuring the `entity_id` property. The template can return a list of entities. The template will only be evaluated at configuration time. Only [limited templates](/docs/configuration/templating/#limited-templates) are allowed.